Readme.md

```diff
@@ -1,13 +1,29 @@
-# Scanbot SDK for Linux
 
-These examples show how to integrate the [Scanbot SDK for Linux](https://scanbot.io) into C++ and Python applications.
+![Scanbot SDK Logo](.images/ScanbotSDKLogo.png)
 
-**The Linux SDK is currently in closed beta.** The SDK and a trial license are available on request: [Get in contact](https://scanbot.io/developer/linux-barcode-scanner/)
+# Example App for the Scanbot Linux Barcode Scanner SDK (Beta)
 
-Refer to the respective README files in the examples directory for more details.
+This example app demonstrates integrating the Scanbot Linux Barcode Scanner SDK into C++ and Python applications.
 
-## Barcode recognition
-The Scanbot SDK allows for the recognition of various barcode types in images and video streams. The SDK is available for both the arm64 and x86_64 architectures. Furthermore, it supports GPU acceleration on NVidia Jetson devices.
+## What is the Scanbot SDK?
+
+The [Scanbot SDK](https://scanbot.io/?utm_source=github.com&utm_medium=referral&utm_campaign=dev_sites) is a set of high-level APIs that integrates barcode, document scanning and data extraction functionalities into mobile apps, websites, and software server for environments, embedded systems, and edge devices. It operates entirely on-device, and no data is transmitted to our or third-party servers.
```

Comment:
We also have document scanner/dqa example for java, not sure if it should be mentioned here or not